home/settings/base.py

```diff
@@ -1,7 +1,8 @@
 import os
 from decouple import config
 
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+BASE_DIR = os.path.dirname(os.path.dirname(
```

Comment:
I think we need to fix our editor configuration to handle longer lines, I am using it with 88 lines, but I have conflict within my vscode that was not able to fix till now,
I will add the rulers per commit :+1: